libs/RobotKit/include/RobotController.h

```diff
@@ -339,7 +339,7 @@ class RobotController : public interface::RobotController
 		auto is_autonomous_mode		= state_machine.is(system::robot::sm::state::autonomous_activities);
 		auto NOT_is_autonomous_mode = !is_autonomous_mode;
 
-		if (card == MagicCard::emergency_stop) {
+		if (card == MagicCard::emergency_stop && rtos::Kernel::Clock::now() - init_timestamp > 10s) {
```

Comment:
separate it in two checks as they are not relevant at the same time:

```suggestion
		if (rtos::Kernel::Clock::now() - init_timestamp < 10s) {
			return;
		}
		
		if (card == MagicCard::emergency_stop) {
```